packages/fxa-shared/oauth/constants.js

```diff
@@ -2,12 +2,15 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+const OAUTH_SCOPE_SUBSCRIPTIONS =
+  'https://identity.mozilla.com/account/subscriptions';
+
 module.exports = {
   OAUTH_SCOPE_OLD_SYNC: 'https://identity.mozilla.com/apps/oldsync',
   OAUTH_SCOPE_SESSION_TOKEN: 'https://identity.mozilla.com/tokens/session',
   OAUTH_SCOPE_NEWSLETTERS: 'https://identity.mozilla.com/account/newsletters',
-  OAUTH_SCOPE_SUBSCRIPTIONS:
-    'https://identity.mozilla.com/account/subscriptions',
+  OAUTH_SCOPE_SUBSCRIPTIONS,
+  OAUTH_SCOPE_SUBSCRIPTIONS_IAP: `${OAUTH_SCOPE_SUBSCRIPTIONS}:iap`,
```

Comment:
I was wrong originally about the `:` for url scopes, it should be `/` or `#`

https://github.com/mozilla/fxa/blob/main/packages/fxa-shared/oauth/scopes.js#L518